### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: _config/extensions.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 Name: cwpcoreextensions
 ---
-# @todo Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
+# Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
```
Comment (sabina-talipova) [inline _config/extensions.yml]: Open ticket: https://github.com/silverstripe/cwp-core/issues/41

File: src/Extension/CustomHtmlEditorField.php [hunk 2]
```diff
@@ -6,10 +6,6 @@
 use SilverStripe\Forms\Form;
 use SilverStripe\View\Requirements;
 
-/**
- * @todo rename file to match class name
- * @todo investigate updated HTMLEditorField_Toolbar changes, and find new implementation
- */
```
Comment (sabina-talipova) [inline src/Extension/CustomHtmlEditorField.php:12]: Open ticket: https://github.com/silverstripe/cwp-core/issues/127

File: src/Extension/LoginAttemptNotifications.php [hunk 3]
```diff
@@ -10,8 +10,6 @@
 use SilverStripe\View\Requirements;
 
 /**
- * TODO: describe.
```
Comment (sabina-talipova) [inline src/Extension/LoginAttemptNotifications.php:13]: Open ticket: https://github.com/silverstripe/cwp-core/issues/127

File: src/Feed/CwpAtomFeed.php [hunk 4]
```diff
@@ -6,7 +6,6 @@
  * CwpAtomFeed class
  *
  * This class is used to create an Atom feed.
- * @todo Improve documentation
```
Comment (sabina-talipova) [inline src/Feed/CwpAtomFeed.php:9]: Open ticket: https://github.com/silverstripe/cwp-core/issues/127

File: src/Extension/LoginAttemptNotifications.php
Comment (sabina-talipova) [inline src/Extension/LoginAttemptNotifications.php:1]: LoginAttemptNotifications extension is currently disabled.

File: _config/extensions.yml [hunk 5]
```diff
@@ -16,7 +16,6 @@ SilverStripe\Security\Group:
   extensions:
     - CWP\Core\Extension\CwpHtmlEditorConfig
 
-# @todo investigate replacement for this
```
Comment (sabina-talipova) [inline _config/extensions.yml:19]: Open ticket: https://github.com/silverstripe/cwp-core/issues/128

File: _config/extensions.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 Name: cwpcoreextensions
 ---
-# @todo Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
+# Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
 # request instead of logins!
 # Follow up issue: https://github.com/silverstripe/cwp-core/issues/41
 # SilverStripe\Admin\LeftAndMain:
```
Comment (ChloeSartorelli) [inline _config/extensions.yml]:
```suggestion
```
Delete the whole todo comment and the related commented-out code please.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: _config/extensions.yml (hunk 6 above)
Comment (sabina-talipova) [reply _config/extensions.yml]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**